data/json/npcs/talk_tags_chat.json

```diff
@@ -69,39 +130,170 @@
+      "Ever hear about those places that supposedly have a natural immunity to the virus? Wonder if they're real.",
+      "Do you think we'll ever find a place where we can just... be safe? Truly safe?",
+      "Sometimes I wonder if this is some kind of test. Like, we're being judged on how we handle it.",
+      "You ever think we'll see the sun rise on a world that's not falling apart?",
+      "Do you think the oceans are safe? Or is there stuff down there now, too?",
+      "What if the cause of all this is still out there, waiting to finish the job?",
+      "You think we'll ever get to a point where we can relax again? Not just survive, but live?",
+      "I used to think the end of the world would bring people together. Guess I was wrong.",
+      "Ever wonder if the animals have it better than us now? Less to worry about, just surviving.",
+      "I sometimes think about all the places I wanted to visit. Wonder what they're like now.",
+      "You ever feel like the whole world is just... tired? Like it's ready to give up?",
+      "Do you think we'll ever have a moment of peace? Just one day where we don't have to worry.",
+      "You ever think about all the things we took for granted? Running water, electricity, safety.",
+      "Sometimes I wonder if the world ended for a reason. Like, maybe we needed a reset.",
+      "Do you think there's anyone out there who knows the full story? Someone with all the answers?",
+      "You ever hear about those secret bunkers? Bet they're laughing at us from their safe havens.",
+      "Ever think about how weird it is that we're still here, fighting to survive? Like, why us?",
+      "I miss the taste of fresh food. Think we'll ever have that again?",
+      "You think we'll ever find a way to make sense of all this? Or are we just stumbling in the dark?",
+      "I used to be scared of the dark. Now it's the least of my worries.",
+      "Do you think there's any hope left? Or are we just clinging to a lost cause?",
+      "Sometimes I wonder if we're just delaying the inevitable. Like, what's the point?",
+      "You ever feel like the whole world is one big haunted house now?",
+      "I can't help but think there's more to this than we know. Like, some big secret waiting to be uncovered.",
+      "You think we'll ever laugh again? Like, really laugh, without any fear or worry?",
+      "Do you think there's a place where people have rebuilt? Like, a hidden utopia?",
+      "Sometimes I wonder if the stars look different now. Like, did they change too?",
+      "I used to think the world was big. Now it feels so small, so fragile.",
+      "You think we'll ever hear music again? Real music, not just the sound of our own fears?",
+      "Ever wonder if the plants and trees know what's happening? If they feel it too?",
+      "Do you think we'll ever see a world without fear again? A place where kids can play outside?"
```

Comment:
```suggestion
      "Remember some of the crazy news from the end of the world?  The stuff that got drowned out by the riot coverage I mean.  Like, didn't the governor of Rhode Island secede from the Union or something?",
      "I keep having dreams that zombies still remember who they were as people, and are just trapped inside the bodies watching everything happen.  Haven't been sleeping well.",
      "Those mind-control drugs they put in the water to make people riot… you think they're still in there?",
      "Do you think there are still secret labs out there, doing god knows what?",
      "Ever wonder if there are places out there that haven't been touched by all this?  Little pockets of normality?",
      "I've heard rumors about entire towns just vanishing overnight.  No bodies, no signs of struggle.  Creepy, huh?",
      "What do you think happened to all the rich and powerful?  Think they had a plan for this?",
      "I wonder if we'll ever see real civilization again, or if this is just the new normal.",
      "Ever think about how the animals are coping with all this?  Mutants are one thing, but what about the regular ones?",
      "Sometimes I think about those survivalist nutjobs.  Bet they're having the last laugh now.",
      "You think any of those doomsday bunkers actually worked?  Might be folks underground right now, clueless about what's happened.",
      "I wonder if there are other dimensions where things didn't go to hell.  Parallel worlds where everything's fine.",
      "There's so much we don't know about what caused this.  You think we'll ever get answers?",
      "I've heard whispers about groups out there, still trying to control things.  Like some shadow government or something.",
      "Some say the world ending was nature's way of hitting the reset button.  What do you think?",
      "You ever think about all the tech we lost?  I mean, we were so close to AI, now we're just scraping by.",
      "I can't stop thinking about those conspiracy theories.  Aliens, government experiments gone wrong...what if some of it's true?",
      "Do you think the military's still out there somewhere?  Holed up, planning a comeback?",
      "I heard a story about a place where the dead don't rise.  Sounds too good to be true, right?",
      "People used to laugh at preppers.  Who's laughing now, huh?",
      "You ever hear about that weird cult that was stockpiling supplies?  Bet they're feeling pretty smug now.",
      "Think any famous people made it through all this?  Celebrities hiding out somewhere?",
      "Remember those stories about underground cities?  Wonder if anyone's living down there now.",
      "You think those sci-fi movies got anything right about how this would go down?",
      "Ever wonder if we're just in some kind of experiment?  Like, some twisted lab rat scenario?",
      "I used to think the apocalypse would be more... glamorous, you know?  Less dirt, more Mad Max.",
      "Do you think we'll ever rebuild?  Like, really rebuild?",
      "I miss the simple things. Like a hot shower.  Think we'll ever have that again?",
      "You ever wonder if the scientists who caused this are still out there, hiding somewhere?",
      "You think there are any places left untouched by all this?  Like, some hidden paradise?",
      "Sometimes I wonder if this is all some sort of punishment.  But for what, who knows?",
      "Ever think about what you'd do differently if you knew this was coming?",
      "I used to dream about adventures.  This... this isn't what I had in mind.",
      "Do you think there are still radio stations out there, broadcasting to no one?",
      "I can't shake the feeling that someone's watching us.  Like, we're not alone in this.",
      "You think we'll ever find a cure for the zombification?  Or is this it for humanity?",
      "Remember when people thought the internet would last forever?  Feels like a joke now.",
      "You think there are still places where people live normal lives?  Kids going to school, adults going to work?",
      "What if there are survivors out there who have no idea what happened?  Completely cut off from everything.",
      "I sometimes wonder if it's better to not know the full truth about what caused all this.",
      "Ever think about how future generations will look at us?  If there even are future generations.",
      "Do you think we'll ever understand the full extent of what happened?  All the whys and hows?",
      "I miss the little things, like traffic jams and crowded malls.  Never thought I'd say that.",
      "You think there's a chance this was all just one big mistake?  Some accident that spiraled out of control?",
      "Ever hear about those places that supposedly have a natural immunity to the virus?  Wonder if they're real.",
      "Do you think we'll ever find a place where we can just... be safe?  Truly safe?",
      "Sometimes I wonder if this is some kind of test.  Like, we're being judged on how we handle it.",
      "You ever think we'll see the sun rise on a world that's not falling apart?",
      "Do you think the oceans are safe?  Or is there stuff down there now, too?",
      "What if the cause of all this is still out there, waiting to finish the job?",
      "You think we'll ever get to a point where we can relax again?  Not just survive, but live?",
      "I used to think the end of the world would bring people together.  Guess I was wrong.",
      "Ever wonder if the animals have it better than us now?  Less to worry about, just surviving.",
      "I sometimes think about all the places I wanted to visit.  Wonder what they're like now.",
      "You ever feel like the whole world is just... tired?  Like it's ready to give up?",
      "Do you think we'll ever have a moment of peace?  Just one day where we don't have to worry.",
      "You ever think about all the things we took for granted?  Running water, electricity, safety.",
      "Sometimes I wonder if the world ended for a reason.  Like, maybe we needed a reset.",
      "Do you think there's anyone out there who knows the full story?  Someone with all the answers?",
      "You ever hear about those secret bunkers?  Bet they're laughing at us from their safe havens.",
      "Ever think about how weird it is that we're still here, fighting to survive?  Like, why us?",
      "I miss the taste of fresh food.  Think we'll ever have that again?",
      "You think we'll ever find a way to make sense of all this?  Or are we just stumbling in the dark?",
      "I used to be scared of the dark.  Now it's the least of my worries.",
      "Do you think there's any hope left?  Or are we just clinging to a lost cause?",
      "Sometimes I wonder if we're just delaying the inevitable.  Like, what's the point?",
      "You ever feel like the whole world is one big haunted house now?",
      "I can't help but think there's more to this than we know.  Like, some big secret waiting to be uncovered.",
      "You think we'll ever laugh again?  Like, really laugh, without any fear or worry?",
      "Do you think there's a place where people have rebuilt?  Like, a hidden utopia?",
      "Sometimes I wonder if the stars look different now.  Like, did they change too?",
      "I used to think the world was big.  Now it feels so small, so fragile.",
      "You think we'll ever hear music again?  Real music, not just the sound of our own fears?",
      "Ever wonder if the plants and trees know what's happening?  If they feel it too?",
      "Do you think we'll ever see a world without fear again?  A place where kids can play outside?"
```